setup.py

```diff
@@ -18,6 +18,8 @@
         'aiohttp>=3,<4',
         'jinja2',
         'rlpython',
+        'typing',
+        'typing-extensions',
```

Comment:
@maratori This is not part of the python standard lib?

Reply:
You are right, `typing` is stdlib (I removed it).
But `typing-extensions` isn't.
```
$ docker run --rm python python -c 'import typing_extensions'
Traceback (most recent call last):
  File "<string>", line 1, in <module>
ModuleNotFoundError: No module named 'typing_extensions'
```

Reply:
@maratori the documentation of typing-extensions states 

>Typing Extensions – Backported and Experimental Type Hints for Python

We don't need backports, or do we? And should we use experimental features?

Reply:
We don't use experimental features, only one backported type `Literal`.
It was introduced in [Python 3.8](https://docs.python.org/3/library/typing.html#typing.Literal).
So we have to use `typing-extensions`, see https://github.com/lona-web-org/lona/pull/69/files#diff-95f7c4beb7cedde530d15bdf9b7c86d405985af7b5fd18ad067728b756f80e7bR11.

Reply:
Ah ok! Thanks for the explanation